scm/pr.go

```diff
@@ -61,6 +67,20 @@ type (
 		PrevFilePath string
 	}
 
+	GeneralCommentMetadata struct{}
+
+	ReplyCommentMetadata struct {
+		ParentID int
```

Comment:
keep in mind not every provider uses a numerical identifier, which is why it is important to research all providers when designing a new abstraction

Reply:
done